### DISCUSSION TITLE
Link prefetching in preview mode

### DISCUSSION THREAD

**Discussion by hatsumatsu** [vercel/next.js#23599] [Help]
I'm trying to understand how my next app behaves in [preview mode](https://nextjs.org/docs/advanced-features/preview-mode). 

As far as I understand all requests in preview mode bypass SSG and ISR and are server rendered on request. Looking at the logs at vercel it looks like link prefetching happens in preview mode, too, which results in quite a number of additional (potentially heavy) SSR requests. 

Wouldn't it make sense to disable link prefetching while in preview mode?

Or is there already a way to disable it that I'm not aware of?

**Comment by hatsumatsu** [comment]
Still trying to wrap my head around the use case of prefetching all links in a page when in preview mode.🥸

**Comment by woochica** [comment]
[There's a bit more active thread around the same topic](https://github.com/vercel/next.js/discussions/29058)
